src/tactic/ring2.lean

```diff
@@ -479,7 +479,7 @@ local postfix `?`:9001 := optional
 
 /-- Tactic for solving equations in the language of rings.
```

Comment:
@digama0 IIRC there are more restrictions than this on `ring2`. Do you remember exactly what it supports?

Reply:
It supports only the commutative semiring operations, i.e. no subtraction or division.